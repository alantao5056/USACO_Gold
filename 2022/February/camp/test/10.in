758 925192000
