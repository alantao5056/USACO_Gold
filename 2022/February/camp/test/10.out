461.47900946290540294
